obspy/io/ascii/tests/test_ascii.py

```diff
@@ -475,6 +475,26 @@ def test_write_small_trace(self):
                 self.assertEqual(len(st), 1)
                 self.assertEqual(len(st[0]), num)
 
+    def test_float_sampling_rates_write_and_read(self):
+        """
+        Tests writing and reading Traces with floating point and with less than
+        1 Hz sampling rates.
+        """
+        tr = Trace(np.arange(10))
+        check_sampling_rates = (0.000000001, 1.000000001, 100.000000001,
+                                99.999999999, 1.5, 1.66666666, 10000.0001)
+        for format in ['SLIST', 'TSPAIR']:
```

Comment:
you are testing double precision values and not float - http://ds.iris.edu/ds/nodes/dmc/data/formats/simple-ascii/ does not speak about double - but it wouldn't hurt to add it as additional datatype